src/argilla/client/feedback/dataset/remote/dataset.py

```diff
@@ -311,6 +319,14 @@ def records(self) -> RemoteFeedbackRecords:
         """
         return self._records
 
+    def update_records(self, records: Union[RemoteFeedbackRecord, List[RemoteFeedbackRecord]]) -> None:
+        if not isinstance(records, list):
+            records = [records]
+
+        # TODO: Use the batch version of endpoint once is implemented
+        for record in records:
+            record.update()
+
```

Comment:
Which is the scenario where someone modifies a `RemoteFeedbackRecord` and then pushes it if not via `RemoteFeedbackRecord.update`? Are we allowing the assignment there? e.g. `record.metadata = {"a": 1}`, if so, won't this be conflictive?